lib/game.rb

```diff
@@ -0,0 +1,64 @@
+class Game
+  attr_accessor :spots, :winner, :draw, :board, :game_on, :first_run, :validator, :current_player
```

Comment:
- [ ] these are the only attr_accessors you need please declare the others as attr_reader

` :current_player, :spots, :first_run, :validator, :game_on`